index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" type="text/css" href="main.css">
+        <title>Sing in project</title>
+    </head>
+
+    <body>
+        <div id="container">
+            <div id="col-1"></div>
+            <div id="col-2">
+                <div id="content-top">
+                    <div>
+                        <img src="./img/intuit_logo.jpg" alt="intuit" id="logo">
+                    </div>
+                    <div id="items">
+                        <ul>
+                            <li><a href="#"> <img src="./img/mint.jpg" alt=""></a> </li>
+                            <li><a href="#"><img src="./img/quickboks.jpg" alt=""></a></li>
+                            <li><a href="#"><img src="./img/turbotax.jpg" alt=""></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div id="form-container">
+                    <div>
+                        <div>
+                            <h1>Create an Intuit account</h1>
+                        </div>
+                        <div>
+                            <p id="txt-1">One account for averything Intuit, Including Ming.
+                        </div>
+                        <div>
+                            <a id="a-1" href="#">Learn more</a></p>
+                        </div>
+
+
+                    </div>
+
+                    <div>
+                        <form idaction="">
+                            <div>
+                                <label for="emailUser" id="input-lb-email">Email address</label>
+                                <div id="input-email">
+                                    <input type="text">
```

Comment:
`REQUIRED`

Consider using `type="email"` for Email address input, this makes adding custom validation and handling of email address more convenient among other dedicated features.

-----


`OPTIONAL`

Consider associating the `<label>` with the `<input>` element, to do this you need to give the `<input>` an id attribute. The `<label>` then needs a for attribute whose value is the same as the input's id. 
- Example 1: 

``` html
  <label for="email">Email address</label> <br>
  <input type="email" name="email" id="email">
```

- Example 2: Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screenreader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)